[2022-08-03 17:00:18,434] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: my_simple_dag.respond 2022-08-03T16:50:00+00:00 [queued]>
[2022-08-03 17:00:18,521] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: my_simple_dag.respond 2022-08-03T16:50:00+00:00 [queued]>
[2022-08-03 17:00:18,565] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 17:00:18,568] {taskinstance.py:1018} INFO - Starting attempt 1 of 1
[2022-08-03 17:00:18,568] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 17:00:18,631] {taskinstance.py:1038} INFO - Executing <Task(PythonOperator): respond> on 2022-08-03T16:50:00+00:00
[2022-08-03 17:00:18,649] {standard_task_runner.py:51} INFO - Started process 20595 to run task
[2022-08-03 17:00:18,667] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'my_simple_dag', 'respond', '2022-08-03T16:50:00+00:00', '--job-id', '1305', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_simple_dag.py', '--cfg-path', '/tmp/tmpmvh7579a']
[2022-08-03 17:00:18,684] {standard_task_runner.py:76} INFO - Job 1305: Subtask respond
[2022-08-03 17:00:19,068] {logging_mixin.py:103} INFO - Running <TaskInstance: my_simple_dag.respond 2022-08-03T16:50:00+00:00 [running]> on host b4dcda255553
[2022-08-03 17:00:20,305] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=my_simple_dag
AIRFLOW_CTX_TASK_ID=respond
AIRFLOW_CTX_EXECUTION_DATE=2022-08-03T16:50:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-03T16:50:00+00:00
[2022-08-03 17:00:20,307] {python.py:118} INFO - Done. Returned value was: Greet Responded Again
[2022-08-03 17:00:20,685] {taskinstance.py:1142} INFO - Marking task as SUCCESS. dag_id=my_simple_dag, task_id=respond, execution_date=20220803T165000, start_date=20220803T170018, end_date=20220803T170020
[2022-08-03 17:00:20,833] {taskinstance.py:1195} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-03 17:00:20,908] {local_task_job.py:118} INFO - Task exited with return code 0
